packages/wallet/wdk/src/sequence/wallets.ts

```diff
@@ -525,6 +526,11 @@ export class Wallets {
     // Sign witness using the passkey signer
     await loginSigner.signer.witness(this.shared.sequence.stateProvider, wallet.address, loginSigner.extra)
 
+    // Sign witness using the guard signer
+    if (guardTopology) {
+      await this.shared.modules.guard.witness(wallet.address)
```

Comment:
I figure the only reason we are signing a witness with the guard is to be able to "know" what kind of signer is the guard signer later, correct?

Reply:
Tbh, I don't know, I thought that every signer must sign a witness, is that not the case? Is it only used for figuring out signer kind? In that case, maybe we don't need it, guard address is static so we'll always know which signer is the guard. Though we can do it "just in case"?

Signing (anything) at this point could be useful to ensure that guard is working and accessible. But I don't know if we can do anything about that anyway.

Reply:
Lets remove this then, and check that everything still works. Lets not do remote calls and sign operations if we don't know what purpose they serve.